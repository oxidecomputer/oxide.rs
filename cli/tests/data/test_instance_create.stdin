{
  "description": "description is required for some reason",
  "disks": [
    {
      "type": "create",
      "description": "boot disk",
      "disk_source": {
        "type": "image",
        "image_id": "42e56270-889a-e74d-fa7c-849b22449cd6"
      },
      "name": "boot",
      "size": 1099511627776
    }
  ],
  "external_ips": [
    {
      "type": "ephemeral"
    }
  ],
  "hostname": "hostname",
  "memory": 4294967296,
  "name": "name",
  "ncpus": 4,
  "network_interfaces": {
    "type": "default"
  },
  "start": true,
  "user_data": ""
}