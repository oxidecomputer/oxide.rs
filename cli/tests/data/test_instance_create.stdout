success
Instance {
    description: "description is required for some reason",
    hostname: "hostname",
    id: 6270889a-e74d-fa7c-849b-22449cd6138e,
    memory: ByteCount(
        4294967296,
    ),
    name: Name(
        "name",
    ),
    ncpus: InstanceCpuCount(
        4,
    ),
    project_id: 78d4484c-6234-dab6-ae7f-7f1572d3814f,
    run_state: Creating,
    time_created: 2018-07-10T22:24:56Z,
    time_modified: 2015-03-05T05:47:06Z,
    time_run_state_updated: 2000-11-09T10:15:15Z,
}
